prov/efa/test/efa_unit_test_ep.c

```diff
@@ -106,17 +106,12 @@ void test_rxr_ep_handshake_exchange_host_id(struct efa_resource **state, uint64_
 	rxr_ep->host_id = g_efa_unit_test_mocks.local_host_id;
 	rxr_ep->use_shm_for_tx = false;
 
-	/* Construct a minimal recv buffer */
-	efa_unit_test_buff_construct(&recv_buff, resource, rxr_ep->min_multi_recv_size);
```

Comment:
Why isn't this necessary anymore?

Reply:
Handshake in the context of EFA protocol happens in the background without the application knowing. It actually does not require `fi_recv/send` - instead it happens purely in the progress engine using prov/efa bounce buffer.

Reply:
Ah got it, yeah EFA automatically posts recv bufs, but if you want to recv it into a user buffer you need to call fi_recv. Can you put the reasoning into the commit description?

Reply:
Good idea. Thanks! Reworded the commit.